### ISSUE TITLE
02. PyTorch Neural Network Classification - activation function plot missing x axis

### ISSUE THREAD

**Issue by fanyewuji** [mrdbourke/pytorch-deep-learning#366]
In Section 7. Replicating non-linear activation functions
![relu](https://user-images.githubusercontent.com/25701581/226905273-af97f6ed-3a04-4397-b9b9-78b64009a380.png)

The relu function plot is missing x axis, so the plot looks a bit strange as it should be 0 when x <=0 and x when x > 0. `plt.plot(A, relu(A))` would be better? The sigmoid function has the same issue. If you like, I can open a PR to fix this
